blueprints/ember-cli-changelog/files/config/changelog.js

```diff
@@ -3,10 +3,10 @@
 // For details on each option run `ember help release`
 module.exports = {
 
+  // ember style guide: https://github.com/emberjs/ember.js/blob/master/CONTRIBUTING.md#commit-tagging
   // angular style guide: https://github.com/angular/angular.js/blob/v1.4.8/CONTRIBUTING.md#commit
   // jquery style guide: https://contribute.jquery.org/commits-and-pull-requests/#commit-guidelines
-  // ember style guide: https://github.com/emberjs/ember.js/blob/master/CONTRIBUTING.md#commit-tagging
-  style: 'angular', // 'ember' 'jquery'
+  style: 'ember', // 'angular' 'jquery'
```

Comment:
To be honest, the default is angular because Ember's style is less used (and less expressive).  Despite being called "angular" style, it's a good format and widely used outside of angular.  I'm ok with making this switch but I'm also hoping to land `log` support soon since it's much harder to pull meaningful commits from Ember style without it.